docsbox/Dockerfile

```diff
@@ -3,7 +3,10 @@ MAINTAINER Dmitry Veselov <d.a.veselov@yandex.ru>
 
 RUN apt-get update && apt-get upgrade -y
 RUN apt-get install -y libffi-dev libmagic-dev libmagickwand-dev
-RUN apt-get install -y libreoffice libreofficekit-dev
+RUN apt-get install software-properties-common -y
+RUN add-apt-repository ppa:libreoffice/ppa
+RUN apt update
+RUN apt install libreoffice -y libreoffice libreofficekit-dev -y
```

Comment:
-y flag is given two times

Reply:
changed the line to:
- apt-get install -y libreoffice libreofficekit-dev